modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -98,16 +99,26 @@ a| `provisioningNetworkInterface` |  | The name of the network interface on node
 
 | `defaultMachinePlatform` | | The default configuration used for machine pools without a platform configuration.
 
-| `apiVIP` | | (Optional) The virtual IP address for Kubernetes API communication.
+| `apiVIPs` | a| (Optional) The virtual IP address for Kubernetes API communication.
+
+This setting must either be provided in the `install-config.yaml` file as a reserved IP from the MachineNetwork or pre-configured in the DNS so that the default name resolves correctly. Use the virtual IP address and not the FQDN when adding a value to the `apiVIPs` configuration setting in the `install-config.yaml` file. The IP address must be from the primary IPv4 or IPv6 network when using dual-stack networking. If not set, the installation program uses `api.<cluster_name>.<base_domain>` to derive the IP address from the DNS.
+
+[NOTE]
+====
+Before {product-title} 4.12, the cluster installation program accepted only an IPv4 address for the `apiVIP` configuration setting. From {product-title} 4.12 or later, the `apiVIP` configuration setting is deprecated. Instead, use a list format for the `apiVIPs` configuration setting to specify an IPv4 addresses, an IPv6 addresses or both IP address formats.
```

Comment:
apiVIP accepted both ipv4 and ipv6 addresses. ipv6 was only for single-stack ipv6 deployments though.